pkg/util/kubernetes/controller/history/controller_history.go

```diff
@@ -0,0 +1,157 @@
+/*
+Copyright 2024 The OpenYurt Authors.
```

Comment:
I think it's more reasonable that copyright belongs to kubernetes.

Reply:
ok

Reply:
reasonable, fixed